packages/server-gateway-interface/package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "@apollo/server-gateway-interface",
+  "version": "0.0.0",
+  "description": "Interface used to connect Apollo Gateway to Apollo Server",
+  "type": "module",
```

Comment:
I think we can drop this given no runtime?
```suggestion
```

Reply:
Are you sure that TSC doesn't look at this field? I guess that's just a nodenext thing? Do you think it hurts?

Reply:
Fair question. I don't think it hurts, and I can't say for sure about `nodenext` so I don't mind if you leave it.